cps/tasks/download.py

```diff
@@ -30,7 +30,6 @@ def __init__(self, task_message, media_url, original_url, current_user_name, she
     def run(self, worker_thread):
         """Run the download task"""
         self.worker_thread = worker_thread
-        log.info("Starting download task for URL: %s", self.media_url)
```

Comment:
> log.info("Starting download task for URL: %s", self.media_url)

Is this line _always_ useless?

(Say why if so?!)

Reply:
The task is added with a similar message.

Reply:
Clarify which code line# (line number) adds the similar message please?

Reply:
https://github.com/iiab/calibre-web/blob/0264fe555da911030adb31342fa55872895fc5c7/cps/tasks/download.py#L159C10-L159C53

Reply:
https://github.com/iiab/calibre-web/blob/e56727ccb456deb19d1ae7a2499130a7c425e51f/cps/services/worker.py#L91